README.md

```diff
@@ -133,7 +133,7 @@ docker container exec -i 12cf98736487 bash
  * [Install the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-linux.html)
  * Build and push watt docker image to AWS repositories and finally restart all WATT instances by
 ```bash
-./watt-aws-instances-update.sh AWS_ACCOUNT_ID
+./watt-aws-instances-update.sh AWS_ACCOUNT_ID [--concurrent-image-push]
```

Comment:
What about changing the option name to `--simultaneous-image-push` or `--parallel-image-push` ?

Reply:
Ok, changed to --simultaneous-image-push.